main/src/bmp.cpp

```diff
@@ -17,7 +17,7 @@ Adafruit_BMP280 bmp_obj;
 void init() {
   if (!bmp_obj.begin(CHIP_ADDR, CHIP_BME)) {
     Serial.println("Viable sensor BMP280 not found, check wiring!");
-    while (1);
+    //while (1);
```

Comment:
why would you comment it out?

Reply:
It was for testing purposes as this line would freeze peogram, we maybe need to think of a solution for testing with missing modules

Reply:
#22